Automattic/pocket-casts-android
# Note will use system font

## Description
Notes in episode show notes will support system fonts

Fixes #618 

## Testing Instructions
1. Turn on a noticeably unusual custom font on your device
2. Open any episode details or full screen player and click notes.
3. Observe the notes font is now using the unusual font you have set as system default font.
## Screenshots or Screencast 
![weird_font](https://user-images.githubusercontent.com/70816971/208289950-04f40fec-0bbf-4c5a-863e-ca07f4ec763e.jpeg)
![another_weird_font](https://user-images.githubusercontent.com/70816971/208289957-c66f0d95-592f-464e-9282-c9f5d88c3d15.jpeg)
![player_notes](https://user-images.githubusercontent.com/70816971/208289964-814d6715-37b2-450b-a53d-c24b1c4234bd.jpeg)

## Checklist
 
#### I have tested any UI changes...
